enforcer/utils/packetgen/interfaces.go

```diff
@@ -0,0 +1,47 @@
+package packetgen
+
+import (
+	"net"
+
+	"github.com/google/gopacket/layers"
+)
+
+type Pkt interface {
+	GenerateIPPacket(srcIPstr string, dstIPstr string) layers.IPv4
+	GenerateTCPPacket(ip *layers.IPv4, srcPort layers.TCPPort, dstPort layers.TCPPort) layers.TCP
+	ChangeSequenceNumber(seqNum uint32) layers.TCP
+	ChangeAcknowledgementNumber(ackNum uint32) layers.TCP
+	ChangeWindow(window uint16) layers.TCP
+	SetSynTrue() layers.TCP
+	SetSynAckTrue() layers.TCP
+	SetAckTrue() layers.TCP
+	ChangePayload(newPayload string) layers.TCP
```

Comment:
Is this Appending if a payload already exists ? 
 - AppendTCPPayload() - what is the return type and why ?
Does it error if a payload exists ?
- NewTCPPayload() and should return error